tracim/tracim/model/data.py

```diff
@@ -544,6 +546,7 @@ class ContentRevisionRO(DeclarativeBase):
     )
     file_mimetype = Column(Unicode(255),  unique=False, nullable=False, default='')
     file_content = deferred(Column(LargeBinary(), unique=False, nullable=True))
```

Comment:
Add a FIXME or TODO for removing this with explanation why (and why it is not yet done)